app/components/application_record_card/style.scss

```diff
@@ -90,10 +126,17 @@
   overflow-wrap: break-word;
 
   span {
-    display: block;
+    @include govuk-media-query($from: $mobile-small-end) {
+      display: block;
+    }
   }
 }
 
+.app-application-trainee_name {
+  margin-bottom: govuk-spacing(1) !important;
```

Comment:
We shouldn't need this `! important`. I suspect it's because of the spacing override used on the heading - if we removed that (and wrote styles to achieve a similar thing, it should work.